test/project_types/extension/commands/register_test.rb

```diff
@@ -6,14 +6,16 @@ module Extension
   module Commands
     class RegisterTest < MiniTest::Test
       include TestHelpers::FakeUI
-      include ExtensionTestHelpers::TempProjectSetup
       include ExtensionTestHelpers::Messages
 
       def setup
         super
         ShopifyCli::ProjectType.load_type(:extension)
-        setup_temp_project(registration_id: nil)
-        @app = Models::App.new(api_key: @api_key, secret: @api_secret)
+        @project = ExtensionTestHelpers.fake_extension_project(with_mocks: true, registration_id: nil)
+        @context = ExtensionTestHelpers.fake_context
```

Comment:
Let's revert this change and rely on `@context` being setup in our `Minitest::Test` override.